hermes-test-helper/src/main/resources/kafkalocal.properties

```diff
@@ -111,3 +111,9 @@ delete.topic.enable=true
 
 # The number of partitions for the offset commit topic.
 offsets.topic.num.partitions=10
+
+# Default replication factor is 3 so we need to customize it
+offsets.topic.replication.factor=1
+
+# Default value is 3000 but there is no need to delay during testing
+group.initial.rebalance.delay.ms = 0
```

Comment:
Are we interested in this config param for prod also?

Reply:
No, this property will be set on broker by Kafka maintenance team